Documentation/gettingstarted/tls-visibility.rst

```diff
@@ -247,7 +247,7 @@ certificate bundle and use it to validate outgoing TLS connections.
 
 .. parsed-literal::
 
-    $ kubectl create secret generic tls-orig-data -n kube-system --from-file=./ca-certificates.crt
+    $ kubectl create secret generic tls-orig-data -n kube-system --from-file=ca.crt=./ca-certificates.crt
```

Comment:
Is that a sort of alias? Where is it used? (I was expecting to find a reference in the policy since you wrote `Use k8s secret key name ca.crt for ca-certificates.crt so that the example CNP works`)

Reply:
In the CNP `terminatingTLS` and `originatingTLS` the optional member `trustedCA` gives the key name in the secret to use, defaulting to `ca.crt`. On `kubectl create secret` `--from-file` option the file name is used as the secret key name by default. This change overrides the default key in the secret to be `ca.crt` instead, which matches the default expected by the TLS context object in the CNP. Alternatively, we could change the CNP to include an explicit `trustedCA` like this:

```
diff --git a/examples/kubernetes-tls-inspection/l7-visibility-tls.yaml b/examples/kubernetes-tls-inspection/l7-visibility-tls.yaml
index 22fcbff9f4..d06b3c17e3 100644
--- a/examples/kubernetes-tls-inspection/l7-visibility-tls.yaml
+++ b/examples/kubernetes-tls-inspection/l7-visibility-tls.yaml
@@ -23,6 +23,7 @@ spec:
         secret:
           namespace: "kube-system"
           name: "tls-orig-data"
+        trustedCA: "ca-certificates.crt"
       rules:
         http:
         - {}
```

Reply:
Thanks for the explanation!